typings/alt/alt.d.ts

```diff
@@ -0,0 +1,158 @@
+// Type definitions for Alt 0.16.7
+// Project: https://github.com/goatslacker/alt
+// Definitions by: Michael Shearer <https://github.com/Shearerbeard>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../flux/flux.d.ts"/>
+/// <reference path="../eventemitter3/eventemitter3.d.ts"/>
```

Comment:
the dependencies it was, i think it is needed i was just confused about the quad slash